src/reducer/videoReducer.jsx

```diff
@@ -0,0 +1,30 @@
+const VideoReducer = (state, action) => {
+    switch (action.type) {
+        case "LOAD_VIDEO":
+            return { ...state, videos: action.payload };
```

Comment:
You don't need {...state} anymore because action.payload will contain the entire video library
Same goes for "LIKE" and "WATCH_LATER"

Reply:
Ok will look into that